deploy/debian-virtualenv/rules

```diff
@@ -12,4 +12,7 @@ override_dh_virtualenv:
 override_dh_shlibdeps:
 	dh_shlibdeps --exclude=numpy --exclude=psycopg2 --exclude=libz --exclude=png16
 
+override_dh_strip:
+	dh_strip --exclude=cffi --exclude=PIL --exclude=Pillow
```

Comment:
i think we should not overuse this and just exclude the Pillow package.

Reply:
Would you be so kind and test this, if "Pillow" is sufficient?

Reply:
If i checked correctly, excluding PIL *and* Pillow will suffice.